CHANGELOG.md

```diff
@@ -3,17 +3,18 @@
 ## [Unreleased]
 
 ### Added
-
+- Added Github Action to verify Gradle wrapper checksum
 ### Changed
-
+- Updated Gradle Wrapper to 8.7
+- Updated Kotlin Version to 1.9.24
+- Update IntelliJ Platform plugin to 1.17.3
```

Comment:
I think it'd be nice if we kept these sections for fixes/features related to plugin functionality.

What do you think about moving these to `## Security` ?